graylog2-web-interface/src/routing/Routes.jsx

```diff
@@ -99,5 +141,9 @@ const qualifyUrls = (routes, appPrefix) => {
 };
 
 const defaultExport = AppConfig.gl2AppPathPrefix() ? qualifyUrls(Routes, AppConfig.gl2AppPathPrefix()) : Routes;
+window.pluginRoutes = AppConfig.gl2AppPathPrefix() ? qualifyUrls(pluginRoutes, AppConfig.gl2AppPathPrefix()) : pluginRoutes;
+
+// Plugin routes need to be prefixed separately, so we add them to the Routes object just at the end.
+defaultExport.pluginRoute = (key) => window.pluginRoutes[key];
```

Comment:
Maybe we should throw an error here if the key does not exist to avoid throwing an error deep down in the stack?